opentasks-provider/src/main/java/org/dmfs/provider/tasks/TaskDatabaseHelper.java

```diff
@@ -40,7 +39,7 @@
 {
 
     /**
-     * Interface of a listener that's called when the database has been created or migrated.
+	 * Interface of a listener that's called when the database has been created or migrated.
```

Comment:
This is strange, I don't see that extra indent in Android Studio. Not in the diff there and not in the editor. But when I open the file in text editor, it's there. It is 2 tabs in plain editor and 1 tab in Android Studio.

Also not sure why even code format replaces spaces with tabs at the first place in this middle line of comment while space is used otherwise.
I could only assume some bug with IntelliJ code formatting, at this point.